ui/desktop/app/styles/app.scss

```diff
@@ -502,7 +502,19 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
   }
 }
 
-.session-details-header,
+.session-details-info {
+  display: flex;
+  align-items: center;
+
+  > * {
+    margin-right: 1rem;
+  }
+
+  .hds-badge {
```

Comment:
As [per this comment](https://github.com/hashicorp/boundary-ui/pull/2018/files#r1414485581), if you apply the approch described there, you do not need to specify a `margin-left` for `hds-badge`